app/scripts/lib/app-start.js

```diff
@@ -130,9 +132,7 @@ function (
         this.initializeInterTabChannel()
       ])
       .then(_.bind(this.allResourcesReady, this))
-      .then(function () {
-        self._trackWindowOnError();
-      }, function (err) {
+      .then(null, function (err) {
```

Comment:
Seems to work, maybe I checked the list last time and thought it does not have it. Thanks!